src/index.html

```diff
@@ -21,7 +24,352 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body body">
+    <header class="body__header header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__links is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__item">
```